go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -293,7 +293,7 @@ func (s *planTestSuite) TestViews() {
 	s.testFile("view_cases.json", vschemaWrapper, false)
 }
 
-func (s *planTestSuite) TestOne() {
+func (s *planTestSuite) /**/ TestOne() {
```

Comment:
Was this change on purpose? 🤔 

```suggestion
func (s *planTestSuite) TestOne() {
```

Reply:
not at all